db/schema_tables.cc

```diff
@@ -1736,6 +1742,14 @@ mutation make_scylla_tables_mutation(schema_ptr table, api::timestamp_type times
     // In-memory tables are deprecated since scylla-2024.1.0
     // FIXME: delete the column when there's no live version supporting it anymore.
     // Writing it here breaks upgrade rollback to versions that do not support the in_memory schema_feature
```

Comment:
Comment looks misplaced now.

Reply:
Can keep it last, after the functional code